OpenEdXMobile/src/main/java/org/edx/mobile/module/analytics/AnswersEvent.java

```diff
@@ -0,0 +1,44 @@
+package org.edx.mobile.module.analytics;
+
+import android.content.res.Configuration;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.text.TextUtils;
+
+import com.crashlytics.android.answers.CustomEvent;
+
+import org.edx.mobile.base.MainApplication;
+
+/**
+ * Wrapper class that defines common functions required for Answers custom event.
```

Comment:
`Utility class that defines a specific format for an analytics event that we deliver to Answers.`